website/docs/r/lb.html.markdown

```diff
@@ -74,6 +74,25 @@ resource "aws_lb" "example" {
 }
 ```
 
+### Specifying private IP addresses for an internal-facing load balancer
+
+```hcl
+resource "aws_lb" "example" {
+  name               = "example"
+  load_balancer_type = "network"
+
+  subnet_mapping {
+    subnet_id     = "${aws_subnet.example1.id}"
+    private_ivp4_address = "10.0.1.15"
```

Comment:
typo: private_ivp4_address vs private_ipv4_address

Reply:
Hello, Is there any chance to get this released ? Many thanks for your review and comments on this, would be really helpfully to see this feature implemented :)